test/zmumu/tp_from_aod_MC.py

```diff
@@ -77,7 +77,8 @@
 process.muonMatchHLTL3.maxDeltaR = 0.1
 from MuonAnalysis.MuonAssociators.patMuonsWithTrigger_cff import *
 changeRecoMuonInput(process, "mergedMuons")
-#useExtendedL1Match(process) #MM no idea what the sequence did, not available since git migration
+#useL1Stage2Candidates(process)
+#appendL1MatchingAlgo(process)
```

Comment:
As the new MC will have the new L1, these 2 lines can be uncommented I guess...
@battibass are you fine with doing it in this PR ?